CMakeLists.txt

```diff
@@ -31,12 +31,14 @@ message(STATUS "Build type is ${CMAKE_BUILD_TYPE}")
 if (WIN32)
 set(POSTGRES_PATHS
   "C:/PostgreSQL"
-  "C:/PostgreSQL/pg96/bin")
+  "C:/PostgreSQL/pg96/bin"
```

Comment:
perhaps at `C:/Program Files/PostgreSQL` as well? Think that is the default location it suggests.